django_app/asrg/settings.py

```diff
@@ -34,9 +34,12 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "app",  # Enable the inner app
     "authentication",
-    "website",
+    "announcements",
+    "job",
+    "landing_page",
+    "contributors",
+    "website",  # TODO: remove after cleanup
```

Comment:
I think this line can be removed

Reply:
I have to run some migrations on prod and only after that I will remove the app and the folder